pydm/config.py

```diff
@@ -19,4 +19,9 @@
 
 STYLESHEET_INCLUDE_DEFAULT = os.getenv("PYDM_STYLESHEET_INCLUDE_DEFAULT", False)
 
-CONFIRM_QUIT = os.getenv("PYDM_CONFIRM_QUIT", "n").lower() in ("y", "t", "1", "true")
\ No newline at end of file
+CONFIRM_QUIT = os.getenv("PYDM_CONFIRM_QUIT", "n").lower() in ("y", "t", "1", "true")
+
+ENTRYPOINT_EXTERNAL_TOOL = "pydm.tool"
+ENTRYPOINT_DATA_PLUGIN = "pydm.data_plugin"
+EXTERNAL_TOOL_SUFFIX = "_tool.py"
+DATA_PLUGIN_SUFFIX = "_plugin.py"
```

Comment:
This is the same suffix we use for the designer plugins- does that matter at all?

Reply:
It's a good question. This mechanism only picks up the classes it's looking for. Designer probably does something similar. I'd imagine if this were problematic, we would likely have gotten bug reports at some point for it?